pkg/file/redundancy/level.go

```diff
@@ -82,6 +82,21 @@ func (l Level) GetMaxEncShards() int {
 	return (swarm.Branches - p) / 2
 }
 
+// GetReplicaCount returns back the dispersed replica number
```

Comment:
These changes expose a lot of convoluted internals to public which is a sign that they are not at the right place.
I think this should remain as per the original, honestly.

Reply:
`GetReplicaCount` is called in hashtrie_test.go and IMO it is like erasure tables.
I moved indexBases back to replicas packages as we discussed offline.